web/src/components/pages/dashboard/stats/leaderboard-card.tsx

```diff
@@ -118,8 +142,14 @@ class UnconnectedLeaderboard extends React.Component<Props, State> {
             {row.position < 9 && '0'}
             {row.position + 1}
           </div>
-          <div className="avatar-container">
+          <div
```

Comment:
Can we make this a button? That would make it more accessible for screen-readers and also reachable with keyboards.

Reply:
Yes! It 's works fine.